diff --git a/module/index.js b/module/index.js
index fe354b7ec4497afba2222b19086fc179f71f3fa6..cfd06a6b9bae8a8a11e5fcbe7c1da1513f30537f 100644
--- a/module/index.js
+++ b/module/index.js
@@ -109,6 +109,7 @@ export const createEngine = async (options) => {
         const loadConfigResult = await loadConfig({
             cwd: options.cwd,
             configFilePath: options.configFilePath,
+            node_moduleDir: path.join(options.cwd, "node_modules"),
         });
         debug("Loaded ConfigFilePath: %s", loadConfigResult.configFilePath);
         return loadConfigResult;
